src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -71,6 +73,16 @@ private void loadData() {
 		listPlaystation.add(playstationGame1);
 		listPlaystation.add(playstationGame2);
 		listPlaystation.add(playstationGame3);
+		
+		//Xbox
+		listXbox = new ArrayList<Xbox>();
+		Xbox xboxGame1 = new Xbox(1, "Halo Infinite", "59.99", "Series X/S");
+		Xbox xboxGame2 = new Xbox(2, "Sunset Overdrive", "19.99", "Xbox One");
+		Xbox xboxGame3 = new Xbox(3, "Forza Horizon 5", "59.99", "Series X/S");
+		
+		listXbox.add(xboxGame1);
+		listXbox.add(xboxGame2);
+		listXbox.add(xboxGame3);
```

Comment:
I created a new list and added some hard coded Xbox data to be displayed on the HTML page I created. I'm not really a gamer, so I'm sorry if the data I added isn't the best :)